code-samples/eventing/helloworld/helloworld-python/requirements.txt

```diff
@@ -1 +1 @@
-Flask==1.1.1
+Flask==2.0.1
```

Comment:
Latest version seems to be [`2.0.3`](https://pypi.org/project/Flask/#history) ... should we go ahead and bump to the latest?

Reply:
Updated, the latest 2.0.3 works also